{
  "data_type": "CVE",
  "data_format": "MITRE",
  "data_version": "4.0",
  "CVE_data_meta": {
    "ID": "CVE-2023-22941",
    "ASSIGNER": "prodsec@splunk.com"
  },
  "problemtype": {
    "problemtype_data": [
      {
        "description": [
          {
            "lang": "en",
            "value": "NVD-CWE-noinfo"
          }
        ]
      }
    ]
  },
  "references": {
    "reference_data": [
      {
        "url": "https://advisory.splunk.com/advisories/SVD-2023-0211",
        "name": "https://advisory.splunk.com/advisories/SVD-2023-0211",
        "refsource": "MISC",
        "tags": [
          "Vendor Advisory"
        ]
      },
      {
        "url": "https://research.splunk.com/application/08978eca-caff-44c1-84dc-53f17def4e14/",
        "name": "https://research.splunk.com/application/08978eca-caff-44c1-84dc-53f17def4e14/",
        "refsource": "MISC",
        "tags": [
          "Vendor Advisory"
        ]
      }
    ]
  },
  "description": {
    "description_data": [
      {
        "lang": "en",
        "value": "In Splunk Enterprise versions below 8.1.13, 8.2.10, and 9.0.4, an improperly-formatted \u2018INGEST_EVAL\u2019 parameter in a Field Transformation crashes the Splunk daemon (splunkd)."
      }
    ]
  }
}